test/suites/container_devices_nic_bridged.sh

```diff
@@ -689,7 +689,14 @@ test_container_devices_nic_bridged() {
   lxc config device get foo eth0 ipv4.address | grep -Fx '192.0.2.233'
   lxc config device get foo eth0 ipv6.address | grep -Fx '2001:db8::3'
   lxc start foo
+
+  # Test container can be copied with device override to fix conflict.
+  lxc copy foo foo-copy --device eth0,ipv4.address=192.0.2.234 --device eth0,ipv6.address=2001:db8::4 --device eth0,hwaddr=0a:92:a7:0d:b7:d8
```

Comment:
Please can you add a test for copying a snapshot with device overrides too.

Reply:
do you think I should include restoring an instances from the copied snapshots and starting them?